roles/engineer.md

```diff
@@ -12,37 +12,14 @@ We need someone:
 
 We expect someone in the role:
 
+- to be billable for at least half their time after 2 weeks on-boarding
```

Comment:
The current way we bill AaaS teams requires a full-time ("good faith") billing commitment. Not to say that's set in stone necessarily, but would require a billing rethink. 

Given where we're currently hiring, do we have an expectation that everyone joining the team could be fully billable in two weeks? I think the difference was when we were focusing our hiring around less experienced engineers?